views/lib/loggedin.hbs

```diff
@@ -252,41 +240,52 @@
-                                {{#unless notitle}}
-                                {{#if title}}
-                                  <div>
-                                      <h1 class="h4" id="page-title" data-testid="{{title}}">{{{stripOnlyScript title}}}</h1>
-                                  </div>
-                                  {{/if}}
-                                {{/unless}}
-                                <div>
-                                    {{subtitle}}
-                                </div>
-                            </div>
-                            {{#if showSearch}}
-                                <div id="searchBar" class="col-sm-3 hide-inline">
-                                    {{#unless liveSearch}}<form class="search-wrapper" action="{{#if searchLabel}}{{searchAction}}{{else}}/content/{{/if}}" method="get">{{/unless}}
-                                        <a href="" class="mobile-search-toggle" aria-label="{{$t "global.lib.loggedin.label.search"}}">
-                                            {{#unless liveSearch}}<i class="fa fa-search"></i>{{/unless}}
-                                        </a>
-                                        <div class="input-group input-group-sm">
-								            <label for="search-box" hidden>{{#if searchLabel}}{{truncatePure searchLabel21}}{{else}}{{$t "lib.loggedin.input.searchFor"}}{{/if}}</label>
-											<input id="search-box" type="text" class="{{#if liveSearch}}js-search{{/if}} form-control search-field {{searchClass}}" placeholder="{{#if searchLabel}}{{truncatePure searchLabel 21}}{{else}}{{$t "lib.loggedin.input.searchFor"}}{{/if}}" name="q" />
-                                            {{> "lib/components/csrfInput"}}
-                                            <span class="input-group-btn">
-                                                <button type="submit" class="btn btn-secondary" aria-label="{{#if searchLabel}}{{truncatePure searchLabel 21}}{{else}}{{$t "lib.loggedin.input.searchFor"}}{{/if}}">
-                                                    <i class="fa fa-search" aria-hidden="true"></i>
-                                                </button>
-                                            </span>
-                                        </div>
-                                    {{#unless liveSearch}}</form>{{/unless}}
-                                </div>
-                            {{/if}}
-                        </div>
-                    </div>
+									<div class="container-fluid">
+										<div class="row" id="titlebar">
+											<div class="col-sm-9">
+												{{#if this.breadcrumbs}}
+													<div style="margin-bottom: 0.4rem; font-size: 14px;">
+														<span>
+															{{#each this.breadcrumbs}}
+																{{#if this.title}}
+																	<a class="{{#if @last}}active{{/if}}" href="{{this.url}}">{{{stripOnlyScript this.title}}}  </a> /
```

Comment:
remove the space before closing </a>
you could add remove the space and `&nbsp; / &nbsp;` but that is also not good.... best is to use css